libs/sdk-ui-kit/src/Dialog/ShareDialog/ShareDialogBase/types.ts

```diff
@@ -126,6 +121,18 @@ export const isGranularGranteeGroup = (obj: unknown): obj is IGranularGranteeGro
     );
 };
 
+/**
+ * @internal
+ */
+export type IGranularGrantee = IGranularGranteeUser | IGranularGranteeGroup;
+
+/**
+ * @internal
+ */
+export const isGranularGrantee = (obj: unknown): obj is IGranularGrantee => {
```

Comment:
This typeguard seems similar to added typaguards in the sdk-model, also what these other types? Should they be also imported, or why to specify them here?

Reply:
From my point of view this is quite unfortunate as the UI component has its own types already defined. These types seem a bit extended and changed for the purposes of the component. That's why the similar typeguards.

Reply:
It was possible to use the other types without changing/copying them so at least `AccessGranularPermission` and `ShareStatus` are used from `sdk-model`. These types just represent some string literals.